tests/unit/test_sql_execution.py

```
@@ -278,6 +278,174 @@ def test_execute_sql_with_connection_json_with_snowflake_encrypted_private_key(
+            [123],
+        )
+
+        sql_alchemy_json = json.dumps(
+            {
+                "url": "trino://user@localhost:8080/catalog",
+                "params": {},
+                "integration_id": "test_integration",
+            }
+        )
+
+        result = execute_sql_with_connection_json(
```

Ruff (0.14.3)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)